gramps/gen/filters/rules/place/_withinarea.py

```diff
@@ -78,6 +78,12 @@ def prepare(self, db, user):
             self.latitude, self.longitude = conv_lat_lon(latitude,
                                                          longitude,
                                                          "D.D8")
+            if self.latitude == None or self.longitude == None:
```

Comment:
Use `is` instead of `==` in comparisons to singletons such as `None`.  See the [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) programming recommendations section.